### PR TITLE
Network page

### PR DESCRIPTION
**Pull request by Flaburgan**
I started to work on the network pages. The code is still quite dirty and will be refactored but I wanted to open this PR to have feedback. As for the previous PR, the texts are not definitive and the design is not correctly done. I just tried to get the data from the DB and to display them. Still a lot of work to do.

### CODE REVIEW

**Comment by Flaburgan** [comment]
So code is still dirty but features should be there. I think this is ready for review, the whole project code has to be cleaned up at the end anyway ;)

File: src/routes.js [hunk 1]
```diff
@@ -28,8 +46,12 @@ routes.info = function (req, res, db) {
 }
 
 routes.renderNetwork = function (network, res, db) {
-  db.Pod.projectStats(network, function (projectStats) {
```
Comment (jaywink) [inline src/routes.js]: So is `db.Pod.projectStats` used anywhere after this? Can be removed in `database.js`?

File: src/routes.js [hunk 2]
```diff
@@ -3,6 +3,24 @@
 var routes = {},
     util = require('util');
 
+var networkTexts = {
```
Comment (jaywink) [inline src/routes.js]: Any chance we could put texts like these in a separate file and `require` them? For example `src/texts.js`.

Comment (Flaburgan) [inline src/routes.js]: I was wondering if we would even put them in the DB so I considered that as a temp. But it can be kept as a json somewhere.

**Comment by Flaburgan** [comment]
Remarks corrected.

File: src/routes.js (hunk 1 above)
Comment (jaywink) [inline src/routes.js]: @Flaburgan I think you were supposed to remove this from database.js by looking at the commit - but it's still there? :)

File: src/texts.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+//TODO Move them directly in the DB?
```
Comment (jaywink) [inline src/texts.js]: Hmm I'm not sure what would be the benefit. If they are in a file we can make them translatable easily in the future. There is no need for these to be dynamic, thus imho they shouldn't be in the db. People can change them via PR's.

**Comment by Flaburgan** [comment]
Corrected.

**Comment by jaywink** [comment]
Alright, thanks a lot, merging!

**Pull request merged**